lib/statsd.js

```diff
@@ -3,7 +3,8 @@ const dgram = require('dgram'),
   dns = require('dns'),
   net = require('net'),
   helpers = require('./helpers'),
-  applyStatsFns = require('./statsFunctions');
+  applyStatsFns = require('./statsFunctions'),
+  unixDgram = require('unix-dgram');
```

Comment:
One thing to consider is that `unix-dgram` isn't supported on non-unix platforms like Windows since it includes `#include <unistd.h>` in the native module (https://github.com/bnoordhuis/node-unix-dgram/blob/master/src/unix_dgram.cc#L9).

If maintaining support for Windows is still desired, you might need to make this an optional dependency and perhaps have a post install script to verify that it was installed correctly for unix operating systems. Unfortunately I can't think of a better idea :)